twins/network.go

```diff
@@ -11,41 +11,58 @@ import (
 
 	"github.com/relab/hotstuff"
 	"github.com/relab/hotstuff/core/logging"
+	"github.com/relab/hotstuff/security/crypto/keygen"
 )
 
 // NodeID is an ID that is unique to a node in the network.
 // The ReplicaID is the ID that the node uses when taking part in the consensus protocol,
 // while the NetworkID is used to distinguish nodes on the network.
 type NodeID struct {
```

Comment:
We should add a doc comment to explain that `TwinID` values are expected. E.g., is `TwinID = 0` ==> no twin, normal replica, while `TwinID = 1` is the first twin, and `TwinID = 2` is the second twin of three replicas?